src/utils/trust/individual.trustee.mapper.ts

```diff
@@ -0,0 +1,46 @@
+import * as Trust from '../../model/trust.model';
+import * as Page from '../../model/trust.page.model';
+import { v4 as uuidv4 } from 'uuid';
+import { RoleWithinTrustType } from '../enum/role.with.trust.type';
+
+export const mapIndividualTrusteeToSession = (
+  formData: Page.IndividualTrusteesFormCommon,
+): Trust.GeneralTrustee => {
+  const data = {
+    trusteeId: formData.trusteeId || uuidv4(),
+    type: formData.type,
+    forename: formData.forename,
+    surname: formData.surname,
+    date_of_birth_day: formData.date_of_birth_day,
+    date_of_birth_month: formData.date_of_birth_month,
+    date_of_birth_year: formData.date_of_birth_year,
+    nationality: formData.nationality,
+    second_nationality: formData.second_nationality,
+    property_name: formData.property_name,
+    address_line1: formData.address_line1,
+    address_line2: formData.address_line2,
+    city: formData.city,
+    county: formData.county,
+    country: formData.country,
+    postal_code: formData.postal_code,
+    correspondence_property_name: formData.correspondence_property_name,
+    correspondence_address_line1: formData.correspondence_address_line1,
+    correspondence_address_line2: formData.correspondence_address_line2,
+    correspondence_city: formData.correspondence_city,
+    correspondence_county: formData.correspondence_county,
+    correspondence_country: formData.correspondence_country,
+    correspondence_postal_code: formData.correspondence_postal_code,
+  };
```

Comment:
The mapped object needs to map to `TrustIndividual` in `Trust.model.ts`. Some fields may need to be added such as second_nationality but others should be mapped directly to existing values where possible. 
Some of the fields are not obvious to what they are(and could do with altering) but for now using the excel spreadsheet and creating a json object with it can help. I can show you what I did.

We need to map to the existing object where possible to minimise effects down stream in the overseas-api and nodesdk for example. If we introduce new fields, objects or alter names extra work will be needed elsewhere.

Reply:
Seconding @JoshFen95 coments on any changes to this model.

Apart from trusteeId (that is only used in the Web application) other fields will need changing in `api-sdk-node`, `overseas-entities-api` and `chips-filing-consumer`. See [Changes to the Historical Beneficial Owners Trustee type](https://companieshouse.atlassian.net/wiki/spaces/TS/pages/4051108053/Changes+to+the+Historical+Beneficial+Owners+BO+Trustee+type) as an example. When changes are requred then we need to get new stories raised to update the other microservices